api/internal/git/repospec.go

```diff
@@ -30,16 +30,16 @@ type RepoSpec struct {
 	// Host, e.g. https://github.com/
 	Host string
 
-	// orgRepo name (organization/repoName),
```

Comment:
I am also not sure where these changes are from. @KnVerey I might need a hand investigating.